tests/installation/isosize.pm

```diff
@@ -4,11 +4,11 @@ use testapi;
 
 sub run {
     my $self   = shift;
-    my $iso    = get_var("ISO");
-    my $size   = -s $iso;
+    my $iso    = get_var("ISO") || get_var('HDD_1');
```

Comment:
Wouldn't this potentially cause problems for other tests that may be using a disk image on HDD_1, but where we dont care about the size of the disk image in those cases?